package.json

```diff
@@ -44,7 +44,8 @@
     "ember-resolver": "^2.0.3",
     "ember-welcome-page": "^1.0.3",
     "eslint-plugin-ember-suave": "^1.0.0",
-    "loader.js": "^4.0.10"
+    "loader.js": "^4.0.10",
+    "phantomjs": "^2.1.7"
```

Comment:
I had to do this for local testing. I'm happy to remove it if you think it's unnecessary.